src/Tag/stories/Tag.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react/types-6-0";
+
+import "./tag.css";
+import { Tag, TagGroup, TagProps } from "../index";
+import { ClockIcon, PhotographIcon } from "../../icons";
+
+export default {
+  title: "Tag",
+  component: Tag,
+} as Meta;
+
+const Component: Story<TagProps> = args => <Tag {...args}>Tag</Tag>;
+
+export const Default = Component.bind({});
+Default.args = { size: "md", onClose: () => alert(1) };
+
+export const GroupArrowNavigation = () => {
+  return (
+    <TagGroup allowArrowNavigation className="flex items-center gap-1">
+      <Tag closable>Tag 1</Tag>
+      <Tag closable>Tag 2</Tag>
+      <Tag closable>Tag 3</Tag>
+    </TagGroup>
+  );
+};
+
+export const GroupNoArrowNavigation = () => {
+  return (
+    <TagGroup className="flex items-center gap-1">
+      <Tag closable>Tag 1</Tag>
+      <Tag closable>Tag 2</Tag>
+      <Tag closable>Tag 3</Tag>
+    </TagGroup>
+  );
+};
+
+export const TagsExample = () => {
+  const [tags, setTags] = React.useState(["One", "Two", "Three"]);
+
+  return (
+    <TagGroup allowArrowNavigation className="flex items-center gap-1">
+      {tags.map(tag => (
+        <Tag
+          closable
+          key={tag}
+          id={tag}
+          onClose={id => setTags(tags.filter(t => t !== id))}
+        >
+          {tag}
+        </Tag>
+      ))}
+    </TagGroup>
+  );
+};
+
+export const PrefixIcon = Default.bind({});
+PrefixIcon.args = { prefix: <ClockIcon /> };
+
+export const SuffixIcon = Default.bind({});
+SuffixIcon.args = { closable: true, suffix: <PhotographIcon /> };
```

Comment:
```suggestion
SuffixIcon.args = { suffix: <PhotographIcon /> };
```

I think we don't want the user to do the `closable="true"` when using custom icon !

Reply:
In this case we should have the closable flag. 

suffix isn't just a custom icon, you can think of it as an Action button.
So even if user provides a custom icon or anything else, it will only be visible if users want the Tag to be closable. 


Let's take this example

```tsx
// would not render CLOSE button
<Tag suffix={<button>CLOSE</button>} />

// would render the CLOSE button
<Tag closable suffix={<button>CLOSE</button>} />
```

The suffix is only for changing the default icon/element not showing it implicitly. 

Material UI, Vercel both also does the same, icon will be only shown if users want the tag to be closable/closed 
https://material-ui.com/api/chip/

![image](https://user-images.githubusercontent.com/35374649/102452009-9f15aa00-405f-11eb-858e-5b840633e937.png)

Reply:
Okay, Got it !